Snakefile

```diff
@@ -414,6 +416,33 @@ rule trinity_stats:
     {TRINITY_HOME}/util/misc/plot_ExN50_statistic.Rscript {output.exN50} &>> {log}
     """
 
+rule busco:
+  """
+  Runs BUSCO to assess the completeness of the transcriptome.
+  """
+  input:
+    transcriptome="transcriptome.fasta"
+  output:
+    out_directory=directory("busco"),
+    report="busco_report.txt"
+  log:
+    "logs/busco.log"
+  params:
+    memory="2"
+  threads:
+    4
+  shell:
+    """
+    lineage={config[lineage]} &> {log}
+    if [ -z "$lineage"] &>> {log}
+    then &>> {log}
+      busco -m transcriptome -i {input.transcriptome} -o {output.out_directory} --auto-lineage -c {threads} &>> {log}
+    else &>> {log}
+      busco -m transcriptome -i {input.transcriptome} -o {output.out_directory} -l $lineage -c {threads} &>> {log}
+    fi &>> {log}
+
+    cp $(ls busco/short_summary*.txt) {output.report} &>> {log}
```

Comment:
Can you do `cp` on multiple files into a file? Isn't it supposed be `cat`?

Reply:
If busco succesfully runs, it produces one text file in busco folder called short_summary.specific.{lineage_name}.busco.txt. If we use the automatic lineage selection, then we don't know in advance the exact filename so I thought to get its name this way and than copy it...

Reply:
I see. Why not just `cp busco/short_summary*.txt {output.report} &>> {log}` ?

Reply:
And perhaps you should check the return code of busco and if it didn't finish correctly, then fail the whole task instead of running the `cp` command.